examples/kubernetes-features/README.md

```diff
@@ -0,0 +1,9 @@
+# Microsoft Azure Container Service Engine - Kubernetes Features
```

Comment:
I changed the path to "kubernetes-config" because all of these settings are under orchestratorProfile.kubernetesConfig in the API model. I thought that was a good trade-off between being too generic (as in "features") vs. being too specific (as in "networking"). Please let me know if you disagree/have a proposal.